dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -373,16 +295,6 @@ public void setRemoveUnusedTags(boolean removeUnusedTags) {
     private String fileSortField;
     private String fileSortOrder;
```

Comment:
These two fields are populated via query string but are in no way interpreted - can be removed